src/CHANGELOG.js

```diff
@@ -7,718 +7,723 @@ import SPELLS from 'common/SPELLS';
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-06-15'),
-    changes: 'Generic: Tier 20 Healing Trinket Implementation',
-    contributors: [Anomoly],
-  },
-  {
-    date: new Date('2017-06-06'),
-    changes: 'Added refresh button to fights list.',
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-29'),
-    changes: 'Fixed a crash when trying to parse a corrupt combatlog.',
-    contributors: [Versaya],
-  },
-  {
-    date: new Date('2017-05-28'),
-    changes: 'Added overhealing percentages to the Cast Efficiency tab.',
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-25'),
-    changes: 'Added Patreon links to the specs I (Zerotorescue) maintain. Please let me know if you think this is inappropriate or makes you hesitate to contribute. Added Discord link.',
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-21'),
-    changes: 'An informative message is now shown when trying to parse a report without combatants (usually due to not having advanced combat logging enabled).',
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-21'),
-    changes: 'Fixed a rare crash when auto attacking something.',
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-20'),
-    changes: 'Added Cooldowns tab to show casts and healing when affected by a cooldown. Added Amalgam\'s Seventh Spine mana gained statistic. Promises no longer includes mana reduction during Innervate.',
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-17'),
-    changes: 'Added Sephuz\'s Secret uptime indicator.',
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-16'),
-    changes: 'Disabled Retribution Paladin spec since it never really came out of the experimental phase.',
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-13'),
-    changes: 'Added full multispec support! The right spec specific parser is now selected based on the spec of the selected person. Only players with supported specs will be displayed in the player selection.',
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-11'),
-    changes: 'Fixed a bunch of bugs. The Always Be Casting/Healing module now supports debuffs which allows me to implement boss Haste buffs to make it more accurate. Elisande\'s Haste buffs are now implemented.',
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-11'),
-    changes: <React.Fragment><b>A lot</b> more changes under the hood in order to make the analyzer multi-spec compatible. Almost everything was changed, so anything might have gotten broken. Please let me know.</React.Fragment>,
-    contributors: [Zerotorescue],
-  },
-  {
-    date: new Date('2017-05-07'),
-    changes: 'Untangled many lines - you shouldn\'t notice a difference.',
-    contributors: [Zerotorescue],
-  },
+    {
```

Comment:
Please set your indentation to the project setting of 2 spaces and revert the indentation changes to this file.